scripts/generate-ckpt.sh

```diff
@@ -65,22 +77,32 @@ if [[ $VERBOSE -eq 1 ]] ; then
     set -x
 fi
 
-if [ -z "$MEMOVERRIDE" ] ; then
-    BASEMEM="$((0x80000000)):$((0x10000000))"
-else
-    BASEMEM=$MEMOVERRIDE
-fi
-SPIKEFLAGS="-p$NHARTS --pmpregions=0 --isa=$ISA -m$BASEMEM"
 BASENAME=$(basename -- $BINARY)
 
 if [ -z "$OUTPATH" ] ; then
-    OUTPATH=$BASENAME.$PC.$INSNS.loadarch
+    OUTPATH=$BASENAME.$((RANDOM)).loadarch
```

Comment:
I don't like generating randomly named output directories. Can we keep the old default naming?

Reply:
The old naming doesn't make sense when there are new options (using a DTS/DTB).  Maybe it can just be a concat of the options used (like `$BASENAME.$PC.loadarch` or `$BASENAME.dtb.loadarch`)

Reply:
Not all the options have to be reflected in the naming ... the `PC/INSNS` is meant to let you quickly generate lots of checkpoints at different PCs, or at different cycle counts.

Reply:
Ok this is updated to a deterministic name (with a bit more qualifiers).